distr_tracing/distr_tracing_rn/distr-tracing-rn-2-9.adoc

```diff
@@ -0,0 +1,196 @@
+
+=== Known issues
+
+* Apache Spark is not supported.
+ifndef::openshift-rosa[]
+
+* The streaming deployment via AMQ/Kafka is unsupported on IBM Z and IBM Power Systems.
+endif::openshift-rosa[]
+
+== {TempoName}
+
+:FeatureName: The {TempoName}
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+=== New features and enhancements
+
+This release introduces the following noteworthy enhancements for the {TempoShortName}:
+
+* Support the link:https://operatorframework.io/operator-capabilities/[operator maturity] Level IV, Deep Insights, which enables upgrading, monitoring, and alerting of `TempoStack` instances and the {TempoOperator}.
+
+* Add Ingress and OpenShift Route configuration for the Gateway.
+
+* Support managed and unmanaged states in the `TempoStack` custom resource.
+
+* Expose the following additional ingestion protocols in the Distributor service: Jaeger Thrift binary, Jaeger Thrift compact, Jaeger gRPC, and Zipkin. When the Gateway is enabled, only the OTLP gRPC is enabled.
+
+* Expose the Jaeger Query gRPC endpoint on the Query Frontend service.
+
+* Support multitenancy without Gateway authentication and authorization.
+
+//=== Technology Preview features
+//not for 2.9
+
+=== Bug fixes
+
+* Before this update, the {TempoOperator} was not compatible with disconnected environments. With this update, the {TempoOperator} supports disconnected environments. (link:https://issues.redhat.com/browse/TRACING-3145[TRACING-3145])
+
+* Before this update, the {TempoOperator} with TLS failed to start on {product-title}. With this update, the mTLS communication is enabled between Tempo components, the Operand starts successfully, and the Jaeger UI is accessible. (link:https://issues.redhat.com/browse/TRACING-3091[TRACING-3091])
+
+* Before this update, the resource limits from the {TempoOperator} caused errors such as *OOMKilled*. With this update, the resource limits for the {TempoOperator} are removed to avoid such errors. (link:https://issues.redhat.com/browse/TRACING-3204[TRACING-3204])
+
+=== Known issues
+
+* Currently, the custom TLS CA option is not implemented for connecting to object storage. (link:https://issues.redhat.com/browse/TRACING-3462[TRACING-3462])
+
+* Currently, when used with the {TempoOperator}, the Jaeger UI only displays services that have sent traces in the last 15 minutes. For services that did not send traces in the last 15 minutes, traces are still stored but not displayed in the Jaeger UI. (link:https://issues.redhat.com/browse/TRACING-3139[TRACING-3139])
+
+* Currently, the {TempoShortName} fails on the IBM Z (`s390x`) architecture. (link:https://issues.redhat.com/browse/TRACING-3545[TRACING-3545])
+
+* Currently, the Tempo query frontend service must not use internal mTLS when Gateway is not deployed. This issue does not affect the Jaeger Query API. The workaround is to disable mTLS. (link:https://issues.redhat.com/browse/TRACING-3510[TRACING-3510])
++
+.Workaround
+:!example-caption:
+====
```

Comment:
This renders in a box because of  `====`. Removing [100](https://github.com/openshift/openshift-docs/pull/62410/files#diff-873c33bdf9e924d854fea381b46c0255c61aa4c7a7692920f3080389d7cdd56fR100) and [130](https://github.com/openshift/openshift-docs/pull/62410/files#diff-873c33bdf9e924d854fea381b46c0255c61aa4c7a7692920f3080389d7cdd56fR130) should fix it

Reply:
Thank you. Applied manually.